app/assets/javascripts/discourse/app/components/flag-action-type.js

```diff
@@ -17,14 +17,9 @@ export default Component.extend({
     return I18n.t("flagging.custom_placeholder_" + nameKey);
   },
 
-  @discourseComputed(
-    "flag.name",
-    "flag.name_key",
-    "flag.is_custom_flag",
-    "username"
-  )
-  formattedName(name, nameKey, isCustomFlag, username) {
-    if (isCustomFlag) {
+  @discourseComputed("flag.name", "flag.name_key", "username")
+  formattedName(name, nameKey, username) {
+    if (["notify_user", "notify_moderators"].includes(nameKey)) {
```

Comment:
Technically this is only needed for `notify_user`, but if we move `notify_moderators` to `client.en.yml` it will break other languages? 🤔